nilearn/datasets.py

```diff
@@ -2799,3 +2800,130 @@ def fetch_abide_pcp(data_dir=None, n_subjects=None, pipeline='cpac',
             files = [np.loadtxt(f) for f in files]
         results[derivative] = files
     return Bunch(**results)
+
+
+def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
```

Comment:
This could go into a seperate PR, but spacenet is the only thing using this fetcher, so yes.